apps/ui/src/components/Modal/Vote.vue

```diff
@@ -99,6 +99,10 @@ const offchainProposal = computed<boolean>(() =>
   offchainNetworks.includes(props.proposal.network)
 );
 
+const isStarknetProposal = computed<boolean>(() =>
```

Comment:
Maybe we should hide only for starknet spaces ?

The snapshot block number should work well on starknet offchain